app/src/main/java/org/jellyfin/mobile/ApplicationModule.kt

```diff
@@ -15,4 +21,14 @@ val applicationModule: Module = module {
     single { OkHttpClient() }
     single(named(PLAYER_EVENT_CHANNEL)) { Channel<PlayerEvent>() }
     single(named(WEBAPP_FUNCTION_CHANNEL)) { Channel<String>() }
+    single {
+        Jellyfin {
+            android(get())
+            appInfo = AppInfo(Constants.APP_INFO_NAME, Constants.APP_INFO_VERSION)
+            logger = TimberLogger()
```

Comment:
Setting the logger should happen before calling the `android` function because of my stupidity in the apiclient